.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Report an issue with the C# standard
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Does the standard disallow code that should be valid? Does the standard allow code that should produce a compiler error? Does the language in the standard leave an ambiguity where the behavior should be mandated?
+
+Include a reference to the clause in the standard containing the violation.
+
+Include the version of C# where you encounter this error. The committee is working on the updated standard for C# 7.3. We have lists for features that will be added in later versions. Bugs that describe changes to behavior that existed in 7.3 are useful. Adding issues for features that aren't covered are less helpful.
+
+**Example**
+Provide an example that shows the problem in the standard. A small code snippet that should be allowed or disallowed, or code that behaves differently than the standard.
```

Comment:
Would it be worth including

```csharp
Add code here
```

or similar (the source of this comment has a triple-backtick fence specifying csharp), to avoid getting badly formatted code?